lib/mpl_toolkits/tests/test_axes_grid1.py

```diff
@@ -226,6 +229,91 @@ def test_inset_axes_without_transform_should_use_parent_axes():
     assert ax.transAxes == ax_ins.transAxes
 
 
+@image_comparison(
+    baseline_images=['fill_facecolor'], extensions=['png'],
+    remove_text=True, style='mpl20')
+def test_fill_facecolor():
+    fig, ax = plt.subplots(1, 5)
+    fig.set_size_inches(5, 5)
+    trans1 = blended_transform_factory(ax[0].transData, ax[0].transData)
```

Comment:
I don't understand this; `blended_transform_factory` is for separate x and y transforms, but you are using the same transform for both, so why not use `ax[0].transData` instead of `trans1` (and the same for the rest)?